pkg/machinery/imager/quirks/quirks.go

```diff
@@ -161,3 +161,15 @@ func (q Quirks) SupportsSELinux() bool {
 
 	return q.v.GTE(minVersionSELinux)
 }
+
+var minTalosVersionSDBootOnly = semver.MustParse("1.10.0")
+
+// TalosVersionSDBootOnly returns true if the Talos version supports only SDBoot for EFI.
+func (q Quirks) TalosVersionSDBootOnly() bool {
```

Comment:
I don't think we need `TalosVersion` in quirks, they are always dependent on Talos verion

Probably `UseSDBootForUEFI` ?